po/r-novice-gapminder.ja.po

```diff
@@ -3536,7 +3536,7 @@ msgstr ""
 # header
 #: r-novice-gapminder/_episodes/02-project-intro.md:120
 msgid "### Separate function definition and application"
-msgstr "### 関数の定義と適応を分けましょう"
+msgstr "### 関数定義と適用は別々にしましょう"
```

Comment:
Updated to reflect changes discussed. Just realised I can edit the branch on gituhb.

https://github.com/masamiy/i18n/blob/a106a54f042ebf174b3c26269bde4faf405494c4/po/r-novice-gapminder.ja.po#L63

Didn't realise I could do that. https://github.com/masamiy/i18n/edit/toc-edit/po/r-novice-gapminder.ja.po?pr=%2Fswcarpentry-ja%2Fi18n%2Fpull%2F114